book/src/unit_tests_dart.md

```diff
@@ -0,0 +1,59 @@
+# Unit tests in dart
+
+To run `flutter` or `dart test` with the bridge you need to load the library on your own development machine (Windows/MacOs/Linux/CI). For that use `loadDylib`, for example:
+```dart
+BridgeImpl initializeExternalLibrary(String path) =>
+    BridgeImpl(
+      loadDylib(path),
+    );
```

Comment:
```suggestion
BridgeImpl initializeExternalLibrary(String path) => BridgeImpl(loadDylib(path));
```

Reply:
good catch, I can fix this in my code as well ;)